kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/notification/NotificationManager.kt

```diff
@@ -216,6 +217,18 @@ class NotificationManager internal constructor(private val context: Context, pri
             }
         }.build()
 
+        mediaSessionConnector.setMediaMetadataProvider {
+            val builder = MediaMetadataCompat.Builder()
+
+            val title = descriptionAdapter?.getCurrentContentTitle(it)
+            if (title != null) builder.putText(MediaMetadataCompat.METADATA_KEY_TITLE, title)
+
+            val artist = descriptionAdapter?.getCurrentContentText(it)
+            if (artist != null) builder.putText(MediaMetadataCompat.METADATA_KEY_ARTIST, artist)
+
+            builder.build()
+        }
+
```

Comment:
We can simplify this by using the `apply` [scope function](https://kotlinlang.org/docs/scope-functions.html) :

```suggestion
        mediaSessionConnector.setMediaMetadataProvider {
            MediaMetadataCompat.Builder().apply {
                val title = descriptionAdapter.getCurrentContentTitle(it)
                putText(MediaMetadataCompat.METADATA_KEY_TITLE, title)

                val artist = descriptionAdapter.getCurrentContentText(it)
                if (artist != null) putText(MediaMetadataCompat.METADATA_KEY_ARTIST, artist)
            }.build()
        }
```

Reply:
Ah very cool, thank you! I've made the change.